langchain/agents/initialize.py

```diff
@@ -56,6 +61,11 @@ def initialize_agent(
         agent_obj = load_agent(
             agent_path, llm=llm, tools=tools, callback_manager=callback_manager
         )
+        try:
+            # TODO: Add tags from the serialized object directly.
+            tags_.append(agent_obj._agent_type)
```

Comment:
Not sure why this isn't giving a lint error. Also fine removing this bit. Also could add in a validator / post init or in the class initializer in case people aren't using this method all the time